en/intro_to_command_line/README.md

```diff
@@ -146,6 +146,24 @@ You'll probably see something similar on your machine. Once you open the command
 
 ---
 
+### Learn more about a command
+
+Many commands you can type at the command prompt have built-in help that you can display and read! For example, to learn more about the Current directory command:
```

Comment:
Is "current" capitalized here intentionally?

Reply:
I had thought the tutorial capitalized it but I think you are right, I was looking at a header... will fix.